grpc/creds/creds_test.go

```diff
@@ -96,8 +103,12 @@ func TestTransportCredentials(t *testing.T) {
 		_ = rawConnB.Close()
 	}()
 
-	conn, _, err = tc.ClientHandshake("A:2020", rawConnC, time.Millisecond)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
I think this was just me mistaking the original timeout when moving to a `context.WithTimeout`.